tos.html

```diff
@@ -8,23 +8,24 @@ <h2 id="1-who-can-use-the-service">1. Who can use the Service</h2>
 </ul>
 <h2 id="2-member-conduct">2. Member Conduct</h2>
 <p>You agree not to use the Service to:</p>
-<ul>
+<ol>
+    <!-- CAUTION: Some of these points are referenced numerically elsewhere in the document. If it's necessary to add points to this list, add to the end if at all possible. -->
     <li>attempt to gain unauthorized access to the Service or to any servers, systems, networks, databases, or other technology or data involved in its provision;</li>
     <li>violate any applicable law or regulation;</li>
     <li>post content which you do not have the right or permission to post, or to infringe any copyright or other proprietary right of any entity or person;</li>
     <li>post content that is hateful, discriminatory, harmful, illegal, or otherwise objectionable;</li>
     <li>impersonate any other person, group, or entity;</li>
-    <li>violate the then-current Code of Conduct of the Service, which is available <a href="https://writing.codidact.com/help/code-of-conduct">in our help center</a>.</li>
+    <li>violate the then-current <a href="https://meta.codidact.com/help/code-of-conduct">Code of Conduct of the Service</a>, which is available in our help center.</li>
 </ul>
```

Comment:
Whoops, this needs to be `</ol>`, else the formatting of the rest of the page is off.  Otherwise looks good to me!